.github/workflows/turf.yml

```diff
@@ -22,14 +22,14 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
+        cache: "pnpm"
         with:
           node-version: ${{ matrix.node-version }}
 
       - name: Install pnpm
-        uses: pnpm/action-setup@v2
+        uses: pnpm/action-setup@v4
```

Comment:
Another new version. v4 will throw an error if there is a mismatch between the action runner pnpm and the one defined in package.json's `packageManager` field. Luckily `version` below is optional and it will use the one in package.json by default 🥳